Doc/install/INSTALL.rst

```diff
@@ -0,0 +1,95 @@
+======================
+Biopython Installation
+======================
+
+This document describes installing Biopython on your computer. To make things as simple as possible, 
+it basically assumes you have nothing related to Python or Biopython on your computer and want to 
+end up with a working installation of Biopython. Biopython supports Mac, UNIX/Linux and Windows machines, but the 
+steps to install biopython vary across the platforms.
+
+
+C Compiler
+==========
+
+1) Unix/Linux
+-------------
+
+GCC is recommende as the C compiler, which is usually available as part of the standard set of packages on any Unix or Linux system.
+
+2) Mac OS X
+------------
+
+Install Apple’s XCode suite from the App Store, and then from the Xcode options also install the optional command line utilities.
```

Comment:
I believe you now do not need the full XCode suite, the command line tools alone is possible.

Reply:
See the XCode instructions we give on http://biopython.org/wiki/Download about running ``xcode-select --install``